docs/sles-installation-guide.md

```diff
@@ -0,0 +1,72 @@
+# Installing Intel® Clear Containers 3.0 on SLES* version 12 SP1
+
+This document _only_ covers installing on a
+[SLES](https://www.suse.com/products/server/) 12 SP1 system. Although,
```

Comment:
Lines 4-6 suggested rewrite: "You can use this guide with more recent Service Packs, but the procedures have not been tested."